mmv1/products/logging/FolderSettings.yaml

```diff
@@ -0,0 +1,84 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderSettings'
+description: |
+  Default resource settings control whether CMEK is required for new log buckets. These settings also determine the storage location for the _Default and _Required log buckets, and whether the _Default sink is enabled or disabled.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Configure default settings for organizations and folders': 'https://cloud.google.com/logging/docs/default-settings'
+  api: 'https://cloud.google.com/logging/docs/reference/v2/rest/v2/TopLevel/getSettings'
+base_url: 'folders/{{folder}}/settings'
+self_link: 'folders/{{folder}}/settings'
+import_format: ['folders/{{folder}}/settings']
+# Hardcode the updateMask since d.HasChanged does not work on create.
+create_url: 'folders/{{folder}}/settings?updateMask=disableDefaultSink,storageLocation,kmsKeyName'
+update_url: 'folders/{{folder}}/settings?updateMask=disableDefaultSink,storageLocation,kmsKeyName'
+# This is a singleton resource that already is created, so create
+# is really an update, and therefore should be PATCHed.
+create_verb: :PATCH
+update_verb: :PATCH
+# update_mask: true
+# This is a singleton resource that cannot be deleted, so skip delete.
+skip_delete: true
+autogen_async: false
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "logging_folder_settings"
+    primary_resource_id: "example"
+    vars:
+      key_name: "k-"
```

Comment:
"BootstrapKMSKey" This is really great info. given the pervasiveness of CMEK across GCP resources, it would be good to add something about this in the magic-modules documentation for adding a resource.

Reply:
created b/315805342 to track this